### PR TITLE
Fix deprecated import warning

### PR DESCRIPTION
**Pull request by Gnouc**
Details: https://github.com/pallets/flask/issues/1135

### CODE REVIEW

File: eve_sqlalchemy/__init__.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 import collections
 from copy import copy
 
-import flask.ext.sqlalchemy as flask_sqlalchemy
+import flask_sqlalchemy as flask_sqlalchemy
```
Comment (dkellner) [inline eve_sqlalchemy/__init__.py]: You can leave out the `as flask_sqlalchemy` in this case.

Comment (Gnouc) [inline eve_sqlalchemy/__init__.py]: @dkellner Ah yes, just do a `sed` without noticing

**Comment by Gnouc** [comment]
@dkellner is it ok now 😃

**Comment by dkellner** [comment]
Thank you! Greetings from Huế!

**Pull request merged**